src/scripts/main.js

```diff
@@ -1,3 +1,25 @@
 'use strict';
 
-// write code here
+const list = document.querySelector('tbody');
+
+const listHeader = [ ...document.querySelectorAll('th') ];
```

Comment:
you can find table header like this -> tbody.tHead
read more about different props connected to tables
https://learn.javascript.ru/dom-navigation#dom-navigation-tables